tool/planet/etcd.go

```diff
@@ -163,9 +152,25 @@ func etcdBackup(backupFile string, backupPrefix []string) (err error) {
 	return nil
 }
 
+func etcdMigrate(fromVersion, toVersion string) (err error) {
+	srcDir := getBaseEtcdDir(fromVersion)
+	dstDir := getBaseEtcdDir(toVersion)
+	log.WithFields(log.Fields{
+		"from": srcDir,
+		"to":   dstDir,
+	}).Info("Copy etcd data.")
+	return trace.Wrap(utils.CopyDirContents(srcDir, dstDir))
+}
+
 // etcdDisable disables etcd on this machine
 // Used during upgrades
-func etcdDisable(upgradeService, stopAPIServer bool) error {
+func etcdDisableUpgrade() error {
```

Comment:
Are these disable/enable upgrade service functions still used anywhere?

Reply:
Not really - will remove incl. the commands using them.